### PR TITLE
chore: portable blst

### LINKED ISSUE #7074: ./lodestar: line 7: 2710 Illegal instruction (core dumped)
### Describe the bug

My lodestar `v1.21.0/ae1f9d5` fails to run after an upgrade. 

```
Sep-09 11:58:43.501[]                 info: Lodestar network=mainnet, version=v1.21.0/ae1f9d5, commit=ae1f9d5d8896cf8e5e1dc42cfeea5fbb4b69e99a
Sep-09 11:58:45.801[]                 info: Connected to LevelDB database path=/home/user/.local/share/lodestar/mainnet/chain-db
Sep-09 11:59:21.984[]                 info: Initializing beacon from a valid db state slot=9920736, epoch=310023, stateRoot=0xace5ae1e69658be542e301a6ef019f0da248da32cce3cfe7406340459e0a7401, isWithinWeakSubjectivityPeriod=true
Sep-09 11:59:24.654[monitoring]       info: Started monitoring service remote=beaconcha.in, machine=ireco, interval=60000
Sep-09 11:59:25.085[chain]            info: Historical state worker started
Sep-09 11:59:25.113[eth1]             info: Eth1 provider urls=http://192.168.1.200:8551,http://192.168.1.201:8551
Sep-09 11:59:25.124[execution]        info: Execution client urls=http://192.168.1.200:8551,http://192.168.1.201:8551
./lodestar: line 7:  2710 Illegal instruction     (core dumped) node --trace-deprecation --max-old-space-size=8192 ./packages/cli/bin/lodestar.js "$@"
```

### Expected behavior

No illegal instruction.

### Steps to reproduce

```
  ./lodestar beacon \
    --network "mainnet" \
    --suggestedFeeRecipient "0xmyAddress" \
    --execution.urls "http://192.168.1.201:8551" \
    --execution.urls "http://192.168.1.200:8551" \
    --jwt-secret "/path/to/jwtsecret.hex" \
    --rest.namespace "*" \
    --rest.address "192.168.1.211" \
    --rest.port "9596" \
    --discv5 \
    --port "9001" \
    --subscribeAllSubnets \
    --monitoring.endpoint 'https://beaconcha.in/api/v1/client/metrics?apikey=FooBarBaz'
```

### Additional context

_No response_

### Operating system

Linux

### Lodestar version or commit hash

ae1f9d5

### PR DESCRIPTION
**Pull request by matthewkeil**
**Motivation**

Switch to portable build of blst-ts

Closes #7074

### CODE REVIEW

**Comment by matthewkeil** [comment]
@wemeetagain this branch was deployed last night to `feat3` and the metrics look commensurate with `unstable`.  Going to let them soak for another 24hours to verify though.  @twoeths let me know if you see anything that jumps out at you.  We changed the way the blst rust code gets compiled in ChainSafe/blst-ts#154

**Comment by wemeetagain** [comment]
So far metrics look good. Equivalent and better metrics in bls job time, peering, etc.

File: package.json [hunk 1]
```diff
@@ -90,4 +90,4 @@
     "vite": "^5.3.4",
     "testcontainers/**/nan": "^2.19.0"
   }
-}
+}
```
Comment (nflaig) [inline package.json]: you have a formatter that removes new lines?

Comment (matthewkeil) [reply package.json]: had issues in the blst release as well... format on save was doing odd things... hmmm looking into it now

Comment (matthewkeil) [reply package.json]: ok, fixed

**Review by nflaig — APPROVED** [review]

**Comment by wemeetagain** [comment]
:tada: This PR is included in [v1.23.0](https://github.com/ChainSafe/lodestar/releases/tag/v1.23.0) :tada:

**Comment by tallero** [comment]
Still fails for me using v1.27.1 from npm.

The build recipe for you to reproduce is at [nodejs-lodestar-ur](https://github.com/termux-pacman/termux-packages/issues/38#issuecomment-1882637215).

Testing building from source now.

**Pull request merged**